src/redisboard/views.py

```diff
@@ -97,66 +97,75 @@ def _get_key_details(conn, db, key, page):
 
 
 def _get_db_summary(server, db):
```

Comment:
Woudln't it be better if you'd rename this function to _raw_get_db_summary (or something) and then just have a wrapper function that calls that and only does the try/except thing? Just to avoid moving around such high amount of code.